app/electron/main.ts

```diff
@@ -386,6 +388,42 @@ function startElecron() {
       mainWindow = null;
     });
 
+    // Deep linked url
+    let deeplinkingUrl;
+
+    // Force Single Instance Application
+    const gotTheLock = app.requestSingleInstanceLock();
+    if (gotTheLock) {
+      app.on('second-instance', (e, argv) => {
+        // Someone tried to run a second instance, we should focus our window.
+
+        // Protocol handler for win32
+        // argv: An array of the second instance’s (command line / deep linked) arguments
+        if (process.platform == 'win32') {
+          // Keep only command line / deep linked arguments
+          deeplinkingUrl = argv.slice(1);
+        }
+        console.log('app.makeSingleInstance# ' + deeplinkingUrl);
+
+        if (mainWindow) {
+          if (mainWindow.isMinimized()) mainWindow.restore();
+          mainWindow.focus();
+        }
+      });
+    } else {
+      app.quit();
+      return;
+    }
+
+    mainWindow.webContents.on('will-navigate', (event, url) => {
+      shell.openExternal(url);
+    });
```

Comment:
Should we check that the URL doesn't have our app's internal base URL one? (in which case we should open it within Electron?)